lib/server/server.js

```diff
@@ -128,6 +128,14 @@ function execute(port) {
   // handle all requests for document pages
   const app = express();
 
+  app.get('/docs', (req, res) => {
+    let firstKey = Object.keys(Metadata)[0];
+    let url = Metadata[firstKey].permalink.includes('docs/en/')
+      ? '/docs/en/'
+      : '/docs/';
+    res.redirect(url + 'installation.html');
```

Comment:
This should not be hardcoded. Not all pages have an installation page and necessarily call it `installation.html`.